Dockerfile

```diff
@@ -7,45 +7,44 @@ RUN pip install poetry
 # copy all except in dockerignore
 COPY . .
 # install prerequisits
-RUN poetry install --no-root --no-dev
-#RUN pip install -r requirements.txt
-# use port 5000 from container
+RUN poetry config virtualenvs.create false --local && poetry install
+# use port from container
 EXPOSE 5000
 
-
-
-# prod image
-FROM base as production
-# install gunicorn
-RUN pip install gunicorn
-# Cmd /entrypoint
-CMD ["gunicorn"  , "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
-
-# dev image
+# development - image
 FROM base as development
 # install flask
 #RUN pip install flask
-
 CMD [ "poetry", "run", "flask", "run", "--host=0.0.0.0", "--port=5000", "--debugger"] 
 
-# testing stage
+# testing Image
 FROM base as test
 RUN pip install pytest
 #RUN poetry Install
-RUN poetry add pytest --dev
+RUN poetry add pytest --group dev
```

Comment:
We shouldn't really be adding this at all here since it should already be installed when you ran the `poetry install` command earlier

Reply:
updated